wandb/sdk/wandb_artifacts.py

```diff
@@ -60,7 +60,7 @@
 
 # This makes the first sleep 1s, and then doubles it up to total times,
 # which makes for ~18 hours.
-_REQUEST_RETRY_STRATEGY = urllib3.util.retry.Retry(
+_REQUEST_RETRY_STRATEGY = requests.packages.urllib3.util.retry.Retry(
```

Comment:
Are we converting to requests from urllib3?

Reply:
I didn't actively make this change, this might have been caused by a merge conflict, I'll change it back to match master.